apps/ehr/tests/e2e/specs/in-person/medicalConditions.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { Page, test } from '@playwright/test';
+import { DateTime } from 'luxon';
+import { CssHeader } from 'tests/e2e/page/CssHeader';
+import { openInPersonProgressNotePage } from 'tests/e2e/page/in-person/InPersonProgressNotePage';
+import { MedicalConditionsPage } from 'tests/e2e/page/MedicalConditionsPage';
+import { SideMenu } from 'tests/e2e/page/SideMenu';
+import { ResourceHandler } from '../../../e2e-utils/resource-handler';
+
+const PROCESS_ID = `medicalConditions.spec.ts-${DateTime.now().toMillis()}`;
+const resourceHandler = new ResourceHandler(PROCESS_ID, 'in-person');
+const MEDICAL_CONDITION = 'Paratyphoid fever A';
+
+test.beforeAll(async () => {
+  if (process.env.INTEGRATION_TEST === 'true') {
+    await resourceHandler.setResourcesFast();
+  } else {
+    await resourceHandler.setResources();
+    await resourceHandler.waitTillAppointmentPreprocessed(resourceHandler.appointment.id!);
+  }
+});
+
+test.afterAll(async () => {
+  await resourceHandler.cleanupResources();
+});
+
+test('MC-1 Medical Conditions. Happy Path', async ({ page }) => {
+  const medicalConditionsPage = await prepareAndOpenMedicalConditions(page);
+  await test.step('MC-1.1 Open Medical Conditions page and Add Medical Condition', async () => {
```

Comment:
am i understanding correctly that at this point the medical conditions page would be open? so the test name should reflect that and be only be about adding the medical condition?